src/path.rs

```diff
@@ -794,6 +812,13 @@ impl PathSeg {
         }
     }
 
+    pub(crate) fn ids(&self) -> Vec<EntityId> {
```

Comment:
not necessary, but we could definitely `impl Iterator<Item=EntityId> for PathSeg`?